streamz/tests/test_kafka.py

```diff
@@ -61,6 +50,7 @@ def stop_docker(name='streamz-kafka', cid=None, let_fail=False):
 
 def launch_kafka():
     stop_docker(let_fail=True)
+    os.system("docker pull spotify/kafka")
```

Comment:
Strictly, docker should do this when you ask to run on the next line, and `os.system` is [discouraged](https://docs.python.org/3.8/library/subprocess.html#replacing-os-system) versus `subprocess.call` here and below.
It would look like (NB: better execute command directly than spawn a shell)
```
subprocess.call(shlex.split("docker pull spotify/kafka"))
```
(if indeed you want to keep the extra line, which is fine)

Reply:
Good point, I didn't know about this. Thanks for pointing this out!